.github/workflows/build_and_package.yaml

```diff
@@ -13,9 +13,42 @@ jobs:
   test:
     uses: chainloop-dev/chainloop/.github/workflows/test.yml@main
 
+  onboard_workflow:
+    name: Onboard Chainloop Workflow
+    needs: test
+    uses: chainloop-dev/labs/.github/workflows/chainloop_onboard.yml@edd4d30e954a4a456dd770f3b7c5e6871c84a661
```

Comment:
I do not think this is needed anylonger, `init` creates the workflow if needed.

Reply:
Oki!